src/modules/form/components/group/TableGroup.tsx

```diff
@@ -0,0 +1,150 @@
+import {
+  Box,
+  Stack,
+  Table,
+  TableCell,
+  TableHead,
+  TableRow,
+  Typography,
+} from '@mui/material';
+
+import { useMemo } from 'react';
+import { GroupItemComponentProps } from '../../types';
+import RequiredLabel from '../RequiredLabel';
+import { ItemType } from '@/types/gqlTypes';
+
+/**
+ * Dynamically renders children in a table. See expected format below.
+ *
+ * The TableHeader labels and helper text will be pulled from the labels defined
+ * on the items in the first "row group".
+ *
+ * {
```

Comment:
Thanks for documenting the structure. Seems like it should meet the needs of the current assessments without much code, which is great!